tools/dds/dds-sniffer/rs-dds-sniffer.cpp

```diff
@@ -59,6 +64,38 @@ struct log_consumer : eprosima::fastdds::dds::LogConsumer
 };
 
 
+static eprosima::fastrtps::rtps::GuidPrefix_t std_prefix;
+std::string print( const librealsense::dds::dds_guid & guid, bool shorten = true )
```

Comment:
shorten unused

Reply:
Used...

Reply:
On what line? I don't see it

Reply:
You're right -- I removed previous code and forgot to put it in the new code, so you found a bug :)